continuous_integration/travis/install.sh

```diff
@@ -19,14 +19,19 @@ conda config --set always_yes yes --set changeps1 no
 conda create -q -n test-environment python=$PYTHON
 source activate test-environment
 
+# Pin matrix items
+touch $HOME/miniconda/envs/test-environment/conda-meta/pinned
```

Comment:
I wrote this before I had written the AppVeyor part and had not updated this. Will happily change to `CONDA_PREFIX`.